src/App.js

```diff
@@ -1,10 +1,10 @@
-import React from 'react'
+import React from "react";
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
 import Homepage from "./pages/Homepage/Homepage";
 import BudgetPage from "./pages/budgetPage/BudgetPage";
 import Expenses from "./components/expenses/Expenses";
-
+import SignInPage from "./pages/signInPage/signInPage";
 
```

Comment:
There is a branch  conflicts in the App.js file

Reply:
Fixed merge conflict.